SOURCES/etc/xapi.d/plugins/smartctl.py

```diff
@@ -5,46 +5,46 @@
 import sys
 import XenAPIPlugin
 
-sys.path.append('.')
+sys.path.append(".")
```

Comment:
Commit "Fix the code to pass the pycodetest"  contains many changes that are not required by `pycodestyle`.

The only error that had to be fixed was: `./SOURCES/etc/xapi.d/plugins/smartctl.py:17:56: E231 missing whitespace after ':'`

Reply:
Fixed